tests/test_checker.py

```diff
@@ -198,10 +213,10 @@ async def test_update_json(self):
 }}""".lstrip()  # noqa: E501
 
         await self._test_update(
-            filename,
-            contents,
-            expected_new_contents,
-            ["Update UnityHubSetup.AppImage to 1.2.3.4"],
+            filename=filename,
+            contents=contents,
+            expected_new_contents=expected_new_contents,
+            expected_updates=["Update UnityHubSetup.AppImage to 1.2.3.4"],
```

Comment:
Please either split unrelated changes to a separate commit or don't include them at all into this PR.
The same applies to other args->kwargs changes below.